packages/locale/src/contexts/index.ts

```diff
@@ -1 +1,2 @@
 export { default as LocaleContext, LocaleContextType, useLocale } from './LocaleContext';
```

Comment:
I'm just curious why we explicitly list all properties in the export instead of using `*`?
Also ,the `default` usage seems redundant since we never use default import...

Reply:
We can do that too, just feel explicit is better. The default will be used once we need to wrap components in the context. 

So we can do something like `<LocaContext.Provider value={value}>{/* children */}</LocaContext.Provider>` instead of the `withTranslations` HOC pattern.

Reply:
Perhaps I didn't express myself correctly. 
If we don't set `default` export in`./LocaleContext`, we can get rid of `default`.
```
export { LocaleContext, LocaleContextType, useLocale } from './LocaleContext';
```
It should be used in the same way with or without `default`.